lib/LedgerSMB/Database.pm

```diff
@@ -545,30 +545,17 @@ sub load_modules {
     return 1;
 }
 
-=head2 $db->load_coa({ country => '2-char-country-code',
-                       chart => 'name-of-chart',
-                       gifi => 'name-of-gifi',
+=head2 $db->load_sic({ country => '2-char-country-code',
                        sic => 'name-of-sic' })
 
 Loads the chart of accounts (and possibly GIFI/SIC tables) as specified in
```

Comment:
Is this comment still correct?
Function name has changed to only encompass SIC....

Reply:
ISO-3166 alpha-2 is currently used. Should we provision for upcoming alpha-3?

Reply:
Do we want/need to move to the 3-char codes? (I'm fine with that, just asking if there's a necessity.)

Note that although these are country codes, they are used as "jurisdiction codes" in the sense that SIC can be different between jurisdictions. (Or, in case of the EU, the same across multiple jurisdictions.)